src/printer.js

```diff
@@ -98,8 +98,7 @@ function genericPrint(path, options, printPath, args) {
 
       if (
         node.decorators.length === 1 &&
-        (decorator.type === "Identifier" ||
```

Comment:
I wasn't sure if this was on purpose. It just looked weird to me that the class keyword wasn't the first on the line. I'm not using them, it if that's how they are formatted I put it back in :)